apisix/plugins/batch-requests.lua

```diff
@@ -45,7 +45,7 @@ local req_schema = {
         timeout = {
             description = "pipeline timeout(ms)",
             type = "integer",
-            default = 30000,
+            default = 3000,
```

Comment:
one PR for one thing. I think you can revert this line and submit it in a new PR

Reply:
or write a summary about what this PR does?

Reply:
> 1. Default timeout in [code](https://github.com/apache/apisix/blob/master/apisix/plugins/batch-requests.lua#L48) is set to 30000. but in [doc](https://github.com/apache/apisix/blob/master/apisix/plugins/batch-requests.lua#L48) it's 3000.

update `doc` is better.

_Originally posted by @membphis in https://github.com/apache/apisix/issues/2262#issuecomment-697154388_

---

Since I turned to update the doc but not the code, I think we could keep them in one PR.
And thank you @juzhiyuan I write a summary. It was too messy without the list.

Reply:
agree: update doc and keep them in one PR. 

pleaes confirm you update the doc and write a right summary ^_^

Reply:
Write the summary in the first block of this PR. Hope I make it clear :)